x-pack/plugin/esql-core/src/main/java/org/elasticsearch/xpack/esql/core/expression/Expression.java

```diff
@@ -26,7 +29,7 @@
  * a, b, ABS(c), and i are all Expressions, with ABS(c) being a Function
  * (which is a type of expression) with a single child, c.
  */
-public abstract class Expression extends Node<Expression> implements Resolvable {
+public abstract class Expression extends Node<Expression> implements Resolvable, NamedWriteable {
```

Comment:
I'm declaring this as `NamedWriteable` now so that I can make `ENTRY` fields on everything that I touch as I go. I can also make tests. But, for the next few weeks, this will be partially implemented as I migrate everything else.